documentation/api/query/v4/packages.markdown

```diff
@@ -0,0 +1,86 @@
+---
+title: "PuppetDB 4.4: Packages endpoint"
+layout: default
+canonical: "/puppetdb/latest/api/query/v4/packages.html"
+---
+
+[curl]: ../curl.html#using-curl-from-localhost-non-sslhttp
+[paging]: ./paging.html
+[query]: ./query.html
+[subqueries]: ./ast.html#subquery-operators
+[facts-format]: ../../wire_format/
+[ast]: ./ast.html
+[pql]: ./pql.html
+[factsets]: ./factsets.html
+[reports]: ./reports.html
+[catalogs]: ./catalogs.html
+[nodes]: ./nodes.html
+[facts]: ./facts.html
+[fact_contents]: ./fact_contents.html
+[events]: ./events.html
+[edges]: ./edges.html
+[resources]: ./resources.html
+[inventory]: ./inventory.html
+
+Package metadata collection, storage, and query is primarily a Puppet Enterprise
+feature. This data is queryable from PuppetDB if it is submitted via the
+`package_inventory` key in the payload of the `store facts` command, as
+described in the [facts format documentation.][facts-format]
+
+## Packages entity (root query endpoint)
+
+Package data is available via the root query endpoint, using the `packages`
+entity name in [PQL][pql] or with a `["from" "packages"]` [AST][ast] query.
+
+### Query fields
+
+* `certname` (string): The certname of the node the package data was collected
+  from.
+
+* `package_name` (string): The name of the package. (for example: `emacs24`)
+
+* `version` (string): The version of the package, in the format used by the
+  package provider. (for example: `24.5+1-6ubuntu1`)
+
+* `provider` (string): The name of the provder which the package data came from;
+  typically the name of the packaging system. (for example: `apt`)
```

Comment:
a nitpick, but the lowercased parentheticals seem weird. What about restyling like

```
The name of the provder which the package data came from; typically the name of the packaging system (e.g 'apt').
```
?